docs/vo/index.rst

```diff
@@ -6,19 +6,20 @@ Virtual Observatory Access (``astropy.vo``)
 
 .. module:: astropy.vo
 
-Introduction
-============
+.. warning::
+
+    The ``astropy.vo`` module has been deprecated and will be removed
+    in a future version. The SAMP functionality has been moved
+    to ``astropy.samp``, and the conesearch functionality has been
+    moved to `Astroquery <http://astroquery.readthedocs.io>`_.
```

Comment:
Oops!  Sorry, misunderstanding: I thought the what's new page was saying that there "classic" API is in astropy, but I get now that that's referencing the astroquery version.  So no change needed.